contrib/compare_ids_test.py

```diff
@@ -38,24 +38,24 @@ def compare_ids(tars, id_=None):
     tars: list of str paths to the image tarballs
     id_: (optional) the id we want the images to have
           if None, just makes sure they are all the same
-
+  Raises:
+    RuntimeError: Expected digest did not match actual image digest
   """
-
   for image in tars:
     current_id = get_id(image)
     if id_ is None:
       id_ = current_id
     elif current_id != id_:
-      exit(1)
-
-  exit(0)
+      raise RuntimeError("Digest mismatch: actual {} vs expected {}".format(
```

Comment:
Why not just print a message before exit(1). But I don't have a strong preference between the two.

Reply:
Personally I prefer exceptions because it gives a full stack when if I need to debug further